docs/guide/obs-integrations.asciidoc

```diff
@@ -0,0 +1,185 @@
+[[observability-integrations]]
+== Observability integrations
+
+Elastic APM supports integrations with other observability solutions.
+
+// remove float tag once other integrations are added
+[float]
+[[apm-logging-integration]]
+=== Logging integration
+
+Many applications use logging frameworks to help record, format, and append an application's logs.
+Elastic APM now offers a way to make your application logs even more useful,
+by integrating with the most popular logging frameworks in their respective languages.
+This means you can easily inject trace information into your logs,
+allowing you to explore logs in the {kibana-ref}/xpack-logs.html[Log app],
+then jump straight into the corresponding APM traces -- all while preserving the trace context.
+
+To get started:
+
+. Enable log correlation
+. Add APM identifiers to your logs
+. Ingest your logs into Elasticsearch
+
+[float]
+==== Enable Log correlation
+
+// temporary attribute for ECS 1.1
+// Remove after 7.4 release
+:ecs-ref: https://www.elastic.co/guide/en/ecs/1.1
+
+You must first enable log correlation in your APM Agents:
```

Comment:
This depends on the agent. The Java agent requires enablement, but the Go agent doesn't since instrumentation is manual.

Reply:
Wouldn't applying the manual instrumentation also count as "enabling"?

Reply:
The next section is "Add APM identifiers to your logs". For Go, there's just one thing to do: you either instrument the logger, or you manually inject trace IDs into log messages.